NuGet.Docs/Contribute/Setting-up-the-NuGet-Development-Environment.md

```diff
@@ -12,7 +12,7 @@ After VS2015 has been installed, you would need to:
 To build NuGet.VisualStudioExtension, execute the following steps:
 
 1. Add the directory of msbuild 14, e.g. C:\Program Files (x86)\MSBuild\14.0\Bin, to PATH
-1. Download the latest version of [NuGet.exe](www.nuget.org/nuget.exe), and add its directory to PATH
+1. Download the latest version of [NuGet.exe](http://www.nuget.org/nuget.exe), and add its directory to PATH
```

Comment:
should be https://

Note that this need to change shortly to go to dist.nuget.org